examples/cosmos_v5_stream_example.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+# -*- coding: latin-1 -*-
+"""
+cosmos_v5_stream_example.py
+"""
+import argparse
+import hashlib
+import os
+import logging
+
+# See cosmosc2/docs/environment.md for environment documentation
+
+os.environ["COSMOS_VERSION"] = "1.1.1"
```

Comment:
Hopefully this variable isn't required.

Reply:
No that allows you to track a version in the requests.